logs/quic.rst

```diff
@@ -0,0 +1,99 @@
+====
+QUIC
+====
+
+.. versionadded:: 6.1
+
+The QUIC protocol integrates encryption, stream multiplexing and flow control at
+the transport layer. QUIC uses TLS 1.3 by default. The QUIC spicy analyzer
+attempts to handle the QUIC INITIAL packet payload to provide greater
+observability into the protocol, including the server name.
+
+
+quic.log
+========
+
+An example of a :file:`quic.log`.
+
+.. code-block:: console
+
+    zeek@zeek-6.1:~ zeek -C LogAscii::use_json=T -r chromium-115.0.5790.110-api-cirrus-com.pcap
+    zeek@zeek-6.1:~ jq . quic.log
+
+::
+
+  {
+    "ts": 1692198386.837988,
+    "uid": "CA482y1XJVd3d0RYI7",
+    "id.orig_h": "82.239.54.117",
+    "id.orig_p": 53727,
+    "id.resp_h": "110.213.53.115",
+    "id.resp_p": 443,
+    "version": "1",
+    "client_initial_dcid": "95412c47018cdfe8",
+    "server_scid": "d5412c47018cdfe8",
+    "server_name": "api.cirrus-ci.com",
+    "client_protocol": "h3",
+    "history": "ISisH"
+  }
+
+
+QUIC Info
+=========
+
+:zeek:see:`QUIC::Info` provides further details on the current output of the
+:file:`quic.log`. Current fields include:
+
+- **version**: The version of QUIC that was identified. Note, QUICv2 has not
+  been widely tested.
+
+
+- **client_initial_dcid**: When QUIC initiates a connection it uses Random
+  Number Generators to create the first Destination Connection ID (DCID). This
+  DCID is subsequently used for routing and packet protection by client and
+  server.
+
+
+- **server_scid**: A QUIC supported server responds to a DCID by
+  selecting a Source Connection ID (SCID). This usually occur within the
+  server’s first INITIAL packet. This is typically used by the client in
+  subsequent packets, although the SCID can change to adapt to new network
+  conditions.
+
+
+- **client_protocol**: During the Application-Layer Protocol Negotiation
+  (ALPN) extension, if the ClientHello packet is available, it is extracted.
+  In the example above, we can see that HTTPv3 is the client protocol.
+
+
+- **history**: An experimental feature provides a history of the QUIC
+  protocol usage. In the example above the history outlines:
+
+    + An initial packet from the client - a new connection
+
+
+    + An SSL Client/Server Hello from the client - the start of a
+      SSL/TLS handshake
+
+
+    + An initial packet response from the server - an acknowledgement
+      from the server of the new connection
+
+
+    + An SSL Client/Server Hello response from the server - the
+      selection  of a cipher suite from the options provided by the
+      client
+
+    + A handshake packet - completes the TLS negotiation
```

Comment:
```suggestion
    + A handshake packet from the client (H)
```

I don't think these necessarily complete the negotiation, so would leave it out. Also might be good to avoid mixing SSL and TLS and stick with one of them in the docs.

Reply:
✅